Code-the-Dream-School/dd-prac-team1-front
# PRA-180 Planner - connecting endpoints

##  Planner - connecting endpoints
## Requirements

All endpoints are connected.

## Notes

The planner is not responsive right now. Responsiveness will be added at the last stage.

## Test Steps

You can drag from saved recipes into the planner.
You can't drag back to the saved recipes - the meal drags back to the previous place.
The amount of saved recipes shouldn't change.
You can drag meals inside the planner.
For now you can't replace meals - this feature is under the discussion with the backend.
If you drag and don't drop into the slot, the meal drags back to the previous place.
If you hover over the recipe in the planner, you can see the button and delete it the meal. 
You can press "clear all" and delete all the meals in the planner (not saved recipes).
You can add the meals' ingredients to the shopping list.

## UI/UX Outcome

<img width="1686" alt="Screen Shot 2023-09-03 at 18 40 21" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/e446fab3-8f3a-4557-97ee-534c6bc32c8e">

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
